go.mod

```diff
@@ -171,3 +171,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/crossplane/crossplane-runtime => ./third_party/github.com/crossplane/crossplane-runtime
```

Comment:
You don't need to create a third_party directory to test your changes to crossplane-runtime. You can point this replace statement at your branch. e.g. `=> github.com/diviner524/crossplane-runtime@support-namespace-object`